jenkins/github_helpers.py

```diff
@@ -14,196 +14,186 @@
-    for repo in repos_list:
-        if repo.name == repo_name:
-            return repo
-
-    raise Exception(
-        "Could not connect to the repository: {}. "
-        "Please make sure you are using the correct "
-        "credentials and try again.".format(repo_name)
-    )
-
-
-def branch_exists(repository, branch_name):
-    """
-    Checks to see if this branch name already exists
-    """
-    try:
-        repository.get_branch(branch_name)
-    except:
-        return False
-    return True
-
-
-def get_modified_files_list(repo_root):
-    """
-    Use the Git library to run the ls-files command to find
-    the list of files modified.
-    """
-    git_instance = Git(repo_root)
-    git_instance.init()
-    modified_files = git_instance.ls_files("--modified")
-
-    if len(modified_files) > 0:
-        return modified_files.split("\n")
-    else:
-        return []
-
-
-def create_branch(repository, branch_name, sha):
-    """
-    Create a new branch with the given sha as its head.
-    """
-    try:
-        branch_object = repository.create_git_ref(branch_name, sha)
-    except:
-        raise Exception(
-            "Unable to create git branch: {}. "
-            "Check to make sure this branch doesn't already exist.".format(branch_name)
-        )
-    return branch_object
-
-
-def close_existing_pull_requests(repository, user_login, user_name):
-    """
-    Close any existing PR's by the bot user in this PR. This will help
-    reduce clutter, since any old PR's will be obsolete.
-    """
-    pulls = repository.get_pulls(state="open")
-    deleted_pull_numbers = []
-    for pr in pulls:
-        user = pr.user
-        if user.login == user_login and user.name == user_name:
-            logger.info("Deleting PR: #{}".format(pr.number))
-            pr.create_issue_comment("Closing obsolete PR.")
-            pr.edit(state="closed")
-            deleted_pull_numbers.append(pr.number)
-
-            branch_name = pr.head.ref
-            delete_branch(repository, branch_name)
-    return deleted_pull_numbers
-
-
-def create_pull_request(repository, title, body, base, head, user_reviewers=GithubObject.NotSet,
-                        team_reviewers=GithubObject.NotSet):
-    """
-    Create a new pull request with the changes in head. And tag a list of teams
-    for a review.
-    """
-    try:
-        pull_request = repository.create_pull(
-            title=title,
-            body=body,
-            base=base,
-            head=head
-        )
-        any_reviewers = (
-            user_reviewers is not GithubObject.NotSet or
-            team_reviewers is not GithubObject.NotSet
-        )
-        if any_reviewers:
-            pull_request.create_review_request(
-                reviewers=user_reviewers,
-                team_reviewers=team_reviewers,
+    def branch_exists(self, repository, branch_name):
+        """
+        Checks to see if this branch name already exists
+        """
+        try:
```

Comment:
```
try:
            return repository.get_branch(branch_name)
  except:
            return False
```

Reply:
I think the intention of this method was to return the status as a boolean.